packages/kbn-data-stream-adapter/src/field_maps/ecs_field_map.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EcsFlat } from '@kbn/ecs';
+import type { EcsMetadata, FieldMap } from './types';
+
+const EXCLUDED_TYPES = ['constant_keyword'];
```

Comment:
Consider adding a comment explaining why `constant_keyword` types are excluded

Reply:
That was taken from Alerts data stream implementation, do you know the reason?